### ISSUE TITLE
[Bug] No more coins after restarting Docker

### ISSUE THREAD

**Issue by Janaka-Steph** [vulpemventures/nigiri#110]
Hello,

Environment: 
* MacOS Big Sur
* Docker Desktop 3.1.0 / Engine 20.10.2

When restarting Docker Desktop all resources are running, I don't `nigiri start --liquid`. However I don't have any coins anymore.
```
$ nigiri rpc --liquid getbalance
{
  "bitcoin": 0.00000000
}
$ nigiri rpc getbalance 
0.00000000 
```

Executing the `mint` command results in this error:
```
  Error: Request failed with status code 500
        at createError (/Users/steph/Desktop/vulpem/marina_steph/node_modules/axios/lib/core/createError.js:16:15)
        at settle (/Users/steph/Desktop/vulpem/marina_steph/node_modules/axios/lib/core/settle.js:17:12)
        at IncomingMessage.handleStreamEnd (/Users/steph/Desktop/vulpem/marina_steph/node_modules/axios/lib/adapters/http.js:260:11)
        at IncomingMessage.emit (events.js:327:22)
        at endReadableNT (internal/streams/readable.js:1327:12)
        at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  ...
        config: {
          url: 'http://localhost:3001/mint',
          method: 'post',
          data: '{"address":"el1qqvftsf5vfxke03j73tg3qljzzgc0yndka6hj4cf9qcyg2nvp72rftyyqqj4pfjnpe0du54xfydx8xkf5552m5ajygffmp6hyx","quantity":1}',
         ...
        },
        data: 'Method issueasset failed with error: Insufficient funds\n'
      },
      isAxiosError: true,
      toJSON: [Function: toJSON]
    }
```

I have to run `nigiri stop --delete` and start again each time I restart Docker Desktop.

**Comment by Janaka-Steph** [comment]
Chopsticks logs after restarting Docker:
```
time="2021-02-22T11:53:29Z" level=info msg="Starting server with configuration:" chain=bitcoin electrs_url="http://10.10.0.12:3002" faucet_enabled=true listen_url="0.0.0.0:3000" logger_enabled=true mining_enabled=true rpc_server_url="http://admin1:123@10.10.0.10:19001" tls_enabled=false

time="2021-02-22T11:53:29Z" level=warning msg="Faucet not funded, check the error" error="Post \"http://admin1:***@10.10.0.10:19001\": dial tcp 10.10.0.10:19001: connect: connection refused" status=500

2021/02/22 11:54:55 http: proxy error: dial tcp 10.10.0.12:3002: connect: connection refused

time="2021-02-22T11:54:55Z" level=info msg="New request from remote address: 10.10.0.1:56852" duration=1.851898ms hostname="10.10.0.12:3002" method=GET path=/blocks/ response= start_time="2021-02-22T11:54:55Z" status=502

2021/02/22 11:54:55 http: proxy error: dial tcp 10.10.0.12:3002: connect: connection refused

time="2021-02-22T11:54:55Z" level=info msg="New request from remote address: 10.10.0.1:56852" duration="406.592µs" hostname="10.10.0.12:3002" method=GET path=/blocks/tip/height response= start_time="2021-02-22T11:54:55Z" status=502
```

**Comment by Janaka-Steph** [comment]
Solved in v0.2.0
